config/config.sample.yaml

```diff
@@ -1,17 +1,40 @@
-# Port for incoming slack requests from webhooks and event API messages.
-slack_hook_port: 9898
-# Prefix of incoming requests to strip when handing. This is NOT the bind host
-  #NOTE: Unlike most of the other urls, this one cannot use localhost,
-  #as this one must be publicly visible to the slack API
-inbound_uri_prefix: "https://my.server.here:9898/"
+homeserver: # Required
+  # The domain name of your homeserver
+  server_name: my.server.here
+  # The URL which the bridge server can reach the homeserver on.
+  # You can use localhost so long as the bridge and homeserver are
+  # hosted on the same machine
+  # 8008 is assumed to be the port the homeserver listens on
+  url: http://localhost:8008
+  # The public facing url for media on your homeserver.
+  # This is usually the public url of your homeserver.
+  # Optional, will use `url` by default.
+  # media_url: "http://my.server.here"
+  # Optional. The maximum size of a uploaded file to Matrix in bytes. No limit by default
+  # max_upload_size: 104857600
```

Comment:
Could you either:

* Remove the space before commented out option names or
* Put a newline between options

as I find it hard to notice that there are multiple options in this comment blob.

Reply:
It's a bit easier with syntax highlighting.
All my code editors put a space after the comment character, so it's very annoying to maintain comments without the space.
Adding newlines makes the file length significantly longer and less readable when introducing a newline consistently between all properties.

Reply:
These are just suggestions that we apply to [synapse's config](https://github.com/matrix-org/synapse/blob/develop/docs/sample_config.yaml), which while monumentally longer, we don't receive complaints about the length.

If you don't want to apply them though, that's fine - it's only a suggestion.